docs/en/getting-started.md

```diff
@@ -0,0 +1,125 @@
+## **Getting Started \- Quick Touch with Deployments**
+
+This tutorial will teach you how to visualize the state changes of Kubernetes objects using Kubernetes History Inspector (KHI). Using Deployments as an example, you'll create, scale, roll out, and delete Pods, and then use KHI to see how these operations change the state of Pods and ReplicaSets.
+
+Please see [User Guide](./user-guide.md) as well.
+ 
+### **Prerequisites**
+
+* Available GKE cluster
+* kubectl  
+* Docker or podman   
+* gcloud
+
+<details>
+
+<summary>Quick reference: Create a GKE cluster</summary>
+
+1. Go to Cloud Shell [https://shell.cloud.google.com/](https://shell.cloud.google.com/)
+2. Run gcloud command `gcloud container clusters create khi-tutorial`
+
+</details>
+
+### **Preparation**
+
+KHI is a tool for visualizing past events in a Kubernetes cluster. In other words, KHI cannot be used without historical data in the Kubernetes cluster.  
+First, let's create a Deployment, scale it out, roll it out, and see what it looks like. Also, note the current time as you'll use it later.
+
+1. **Creating a Deployment:**
+
+```
```

Comment:
Maybe we need `gcloud container clusters get-credentials --location <LOCATION> <cluster-name>`

Reply:
you right, good catch!